scripts/test_in_docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Run the Tari test suite locally inside a suitable docker container
+
+IMAGE=quay.io/tarilabs/rust_tari-build-with-deps:nightly-2019-10-04
+TOOLCHAIN_VERSION=nightly-2019-10-04
+CONTAINER=tari_test
+
+echo "Deleting old container"
+docker rm -f $CONTAINER
+#docker pull $IMAGE
+
+echo "Creating docker container.."
+# sleep infinity is used to keep the container alive forever
+docker run -d --name $CONTAINER $IMAGE /bin/sleep infinity
+echo "Container is ready"
+
+echo "Copying source files..."
+# The simplest and laziest approach to not copying 6+GB of irrelevant files to the container
+cargo clean
+# We could pull this from git, but this lets you test local changes
+docker cp . $CONTAINER:/src/
```

Comment:
One option could be to mount the local dir into the docker image for testing.
This is the one liner I use
` docker run --rm -v`pwd`:/app -it quay.io/tarilabs/rust_tari-build-with-deps:nightly-2019-10-04 /bin/bash`

Reply:
You'll then get a polluted `target` folder anyway because they'll be filled with linux binaries

Reply:
`.dockerignore` would be ideal, except it looks like it's ignored in the cp command

Reply:
Could be - I suspect they don't hash to the same file name, so mac/linux binaries won't interfere with each other - haven't tested this though.

Reply:
could be.. lemme test this quick

Reply:
You sold me.